src/components/title.tsx

```diff
@@ -0,0 +1,35 @@
+import * as React from 'react'
+import styled from 'styled-components'
+import * as fonts from '../assets/fonts'
+
+type TitleSize = "small" | "medium" | "large" | "xlarge" | "xxlarge"
+
+interface TitleProps extends React.HTMLProps<JSX.Element> {
+  titleSize?: TitleSize
+}
+
+const titleSizes = {
+  small: '25px',
+  medium: '30px',
+  large: '37px',
+  xlarge: '50px',
+  xxlarge: '72px'
+}
+
+const Title: React.SFC<TitleProps> = props => (
+  <div className={props.className}>
```

Comment:
I used to be into semantic markup, but nowadays I don't think it's worth spending any brain power on. That is to say, if one thinks there's a more appropriate tag for the situation then definitely go ahead and use it but don't spend more than half a second on it and feel free to just use a `div` and move on. As the web gets further away from its simple document viewer days the arguments about semantic markup become weaker, and any onsite SEO tag cleanup efforts I've been a part of at Artsy haven't resulted in measurable SEO payoff. (However, I'm not the expert and Christine and friends in the #seo channel might have better thoughts.)

That said, `h1`–`h6` _is_ a more approriate tag for headers than `p` or `div` so there's one reason to pick it, but if we wanted a cross platform "React Native Web" style component then certainly don't let semantic markup stop use from the wins of reusability.